api/db/changelog/db.changelog-master.xml

```diff
@@ -166,6 +166,7 @@
   <include file="changelog/db.changelog-154-wgs-cromwell-extract-submission.xml"/>
   <include file="changelog/db.changelog-155-add-ras-link-entries.xml"/>
   <include file="changelog/db.changelog-156-billing-buffer-entry-status-index.xml"/>
+  <include file="changelog/db.changelog-157-workspace-add-google-project.xml"/>
```

Comment:
FYI: I just merged a new entry here so the number will have to increase.

Unfortunately we don't have a good process for these kinds of conflicts.

Reply:
merged